anoma/namada-interface
# feat: update zip32 path

Resolves #604 

### Changes
- Updated path
- Left default payment address
- Changed tests runner to wasm_bindgen_test

### How to test
- import mnemonic in extension
- check viewing key in the vault storage
- import mnemonic in the cli
- check if viewing key in cli is the same as the one in the vault storage
